enterprise/frontend/src/embedding-sdk/test/sdk-config-errors.unit.spec.tsx

```diff
@@ -27,16 +27,16 @@ import {
 const METABASE_INSTANCE_URL = "path:";
 const AUTH_PROVIDER_URL = "http://auth-provider/metabase-sso";
 
-const defaultAuthUriConfig = defineEmbeddingSdkConfig({
+const defaultAuthUriConfig = defineEmbeddingSdkAuthConfig({
```

Comment:
same as the previous comment, I think we should rename this to
```suggestion
const defaultAuthConfig = defineEmbeddingSdkAuthConfig({
```